variable/models.py

```diff
@@ -63,6 +63,8 @@ class Variable(models.Model):
         trustworthy if the number of missing values in a given period > null_limit.
     """
 
+    NATURES = [("sum", "sum"), ("average", "average"), ("value", "value")]
```

Comment:
Do you have any examples in mind of variables that might be of "sum" or "average" nature? Just curious what these are needed for

Reply:
You can check the variables defined in the [initial data](https://github.com/ImperialCollegeLondon/paricia/blob/develop/utilities/data/variable_variable.json). Some, like `precipitation` are cumulative (so they use `sum`); others are not cumulative, but typically provided as an average over that time or as an instantaneous value at the time of measurement. In the old system, there was no way of distinguishing between these two, but the initial refactoring done last year required this distinction.

Reply:
Ok thanks. I understand the distinction now. I can see that this PR removes the is_cumulative field from the data in the json, but doesn't replace it with the nature field. Is this intended?

Reply:
`is_cumulative` is left as a property, as is it still a useful check. If you refer to the changes to the initial data, you're right I should have been a bit more careful and replace `is_cumulative=true` with `nature=sum` and `is_cumulative=false` with `nature=value` (as there's no way to distinguish `value` and `average`). I'll do so next. Thanks for pointing this.